pages/api/book/event.ts

```diff
@@ -439,6 +440,44 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     }
   }
 
+  //for Daily.co video calls will grab the meeting token for the call
+  const isDaily = evt.location === "integrations:daily";
+  let dailyEvent;
+  if (!rescheduleUid) {
+    dailyEvent = results.filter((ref) => ref.type === "daily")[0]?.createdEvent;
+  } else {
+    dailyEvent = results.filter((ref) => ref.type === "daily_video")[0]?.updatedEvent;
+  }
+
+  let meetingToken;
+  if (isDaily) {
+    const response = await fetch("https://api.daily.co/v1/meeting-tokens", {
+      method: "POST",
+      body: JSON.stringify({ properties: { room_name: dailyEvent.name, is_owner: true } }),
```

Comment:
This line is throwing an error when testing locally:

![](https://user-images.githubusercontent.com/3504472/135672790-d4e85868-00d7-4ecb-afed-2b74d83e6e50.png)

Reply:
Hi @zomars 

Thanks for sharing! I'm trying to recreate this error locally. Are you still able to book or does it completely not allow you to create a booking?

Reply:
Let me try to re-test

Reply:
I couldn't replicate the error anymore

Reply:
was this resolved?

Reply:
Thanks for the comments everyone! In the latest commit, I added an interface for the daily result and updated the daily event to resolve the type error. I think it runs regardless but this should be a little cleaner.